src/Workspaces/Core/Desktop/Workspace/MSBuild/ProjectFile/ProjectMetadataReference.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Diagnostics;
+using System.Linq;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.Text;
+
+namespace Microsoft.CodeAnalysis.MSBuild
+{
+    /// <summary>
+    /// Represents a reference to another project file.
```

Comment:
I think you mean a reference to metadata, right? This isn't used in project-to-project reference scenarios?